assetman/manifest.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import absolute_import, with_statement
+
 
 import os
 import simplejson as json
```

Comment:
For python3, I think it's fine to use the built in json package instead of simplejson.

Reply:
👍 fixed